### PR TITLE
Integrate this repo with Pangeo Gallery

### PR DESCRIPTION
**Pull request by andersy005**
First, fantastic work! I enjoyed going through the `final_analysis.ipynb` notebook.

Second, I took a stab at integrating this repo with [Pangeo Gallery](http://gallery.pangeo.io/). Things seem to be working fine until the notebook execution. @rabernat, I am running into a problem due to a relative import of a utility script used (util.py). Is it the case that binder-bot launches binder with notebooks only and excludes any other files? What happens to local modules (such as `notebooks/util.py` in this case)? Let me know if I am missing something. 

The full logs of my failed runs reside [here](https://github.com/andersy005/cmip6hack-ocean-bgc/runs/620299090?check_suite_focus=true)

This is the error I am getting:

```python

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-b4048e057a7f> in <module>
     30 # or routines that are too extensive and might otherwise clutter
     31 # the notebook design
---> 32 import util
     33 
     34 if util.is_ncar_host():

ModuleNotFoundError: No module named 'util'
ModuleNotFoundError: No module named 'util'
```

### CODE REVIEW

File: binder-gallery.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: CMIP6 Ocean Biogeochemistry
+description: >-
+  This project examines historical simulations and future projections
+  of ocean carbon uptake and storage in the CMIP6 ensemble,
+  and explore their impacts on marine ecosystems drivers,
+  with a focus on primary productivity.
```
Comment (andersy005) [inline binder-gallery.yaml:7]: @mara-freilich, let me know if this description of the repo is accurate. I copied it from the README. Also, Pangeo Gallery is going to use this [image](https://github.com/andersy005/cmip6hack-ocean-bgc/blob/master/thumbnail.png) as a thumbnail. Let me know if there is a better image we can use as a thumbnail.

**Comment by jbusecke** [comment]
We could easily drop the utils and import the catalog implicitly if that helps?

**Comment by andersy005** [comment]
> We could easily drop the utils and import the catalog implicitly if that helps?

Yeah, this may provide a *temporary* fix for this use case. It would still be nice to get binderbot-cli to work even when there are external/local modules.

File: binder-gallery.yaml (hunk 1 above)
Comment (mara-freilich) [reply binder-gallery.yaml:7]: Here's a slightly edited version: "This project examines historical simulations and future projections of ocean carbon uptake and storage in the CMIP6 ensemble, and explores the impact of carbon dioxide emissions on marine ecosystems drivers, with a focus on primary productivity."

Thanks! I think that's a good image to use as a thumbnail

**Comment by andersy005** [comment]
Closing this because it has gone stale :(

**Pull request closed without merging**